lib/Application.js

```diff
@@ -21,6 +23,7 @@ const DEFAULT_OPTIONS = {
 function Application(options = {}) {
   this.options = Object.assign({}, DEFAULT_OPTIONS, options);
   EventEmitter.decorate(this);
+  this.DOMFetcher = new DOMFetcher(/*Our document metadata listener will be here*/);
```

Comment:
nit: I'd say to include this code when the listener is ready, or mark this line with a `TODO` comment above